NBitcoin.Altcoins/Liquid.cs

```diff
@@ -12,7 +14,12 @@ class LiquidRegtest { }
 		static Liquid()
 		{
 			ElementsParams<Liquid>.PeggedAssetId = new uint256("6f0279e9ed041c3d710a9f57d0c02928416460c4b722ae3457a11eec381c526d");
+			ElementsParams<Liquid>.SignedBlocks = true;
+			ElementsParams<Liquid>.BlockHeightInHeader = true;
```

Comment:
why are those? if they are always true, why even put them?

Reply:
depends on your custom chain config for elements.

Reply:
there is no custom chain. I prefer you remove it.